OculusHeader head
uint8   masterMode
uint8   pingRate
uint8   networkSpeed
uint8   gammaCorrection
uint8   flags
float64 range
float64 gainPercent
float64 speedOfSound
float64 salinity
